### PR TITLE
Moved penlights from starting kit for medical staff to the Medidrobe.

### PR DESCRIPTION
**Pull request by Yatosamasensei**
## About The Pull Request

Removed penlights from Doctor, Paramedic and CMO starting loadout, and moved them to the Medidrobe (4 stock)

Still had to do the gitignore thing.  but after this shouldn't have to change it at all on my end.

## Why It's Good For The Game

No more penlights thrown all over the place in medical because nobody wants them.

## Changelog
:cl:
del: Penlights from paramedic, CMO, and doctor starting loadout
add: 4 Penlights in the Medidrobe
/:cl:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -231,3 +231,8 @@ tools/LinuxOneShot/Database
 tools/LinuxOneShot/TGS_Config
 tools/LinuxOneShot/TGS_Instances
 tools/LinuxOneShot/TGS_Logs
+/.vs/SS13KF/v16/Browse.VC.db
+/.vs/SS13KF/v16/Browse.VC.db-shm
+/.vs/SS13KF/v16/Browse.VC.db-wal
+/.vs/SS13KF/v16/Browse.VC.opendb
+/.vs/slnx.sqlite
```
Comment (jaw-sh) [inline .gitignore:238]: What are these for?

**Review by jaw-sh — CHANGES_REQUESTED** [review]
re: flag cup icon
Make a separate PR with pictures please

File: .gitignore (hunk 1 above)
Comment (Yatosamasensei) [reply .gitignore:238]: Those are files that Visual studio automatically added to the project upon me loading it, not sure how to remove. I tried a couple things to make it so I wouldn't have to change the gitignore file to get them to not be picked up, but none of them worked. I thought of one more thing I can try, and if that works i will redo this

**Comment by Yatosamasensei** [comment]
> re: flag cup icon
> Make a separate PR with pictures please

I didn't realize if I did a PR, and then did another commit it would be appended to the current PR.

**Pull request closed without merging**